docs/getting-started/ios/fastlane-swift.md

```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
 
 🎉
 
+## Get Started (SPM) (Beta)
+
+Swift Package Manager support is included as a preliminary feature that allows the usage of _fastlane_ with other SPM packages.
```

Comment:
```suggestion
Swift Package Manager support is included as a preliminary feature to integrate _fastlane_ with other SPM packages.
```

Reply:
@minuscorp this _seems_ better to me, but does it accurately depict what you're suggesting?

Reply:
_fastlane_ is distributed as a package that allows third-party packages to introduce its usage through SPM. Am I explaining well? 😟

Reply:
This seems like a complicated concept to get across, and my mind is _relatively_ fresh this morning, but I'm lost. Perdón, pero no lo entiendo, a lo mejor es más facil explicar en español primero, y despues en ingles? Me preocupa que alguien, como yo, no lo va a entender.

Reply:
El paquete _fastlane_ se distribuye como un paquete SPM, de tal forma que otros paquetes puedan crear ejecutables que lo integren. 
Así queda algo más claro? Es que en una línea es un poco complicado por cómo es la integración de SPM en si 😪

Reply:
¡Gracias!
```suggestion
Swift Package Manager (SPM) support adds the ability to distribute _fastlane_ as a Swift Package Manager package, which allows other packages to integrate with the toolset.
```
Does that match up with what you're saying?